server/zoom/www/cache/time_estimate_cache.py

```diff
@@ -263,10 +262,8 @@ def _check(self):
         :rtype: bool
         """
         try:
-            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.connect((self._host, 80))
-            s.close()
-            return True
+            r = requests.head(self.host, timeout=.1) # this might be too low.
```

Comment:
Won't know until we have metrics, and I didn't want to spend a bunch of time adding configurability around this number before we have those metrics.